api/v1/stats/index.php

```diff
@@ -26,4 +28,13 @@
 } elseif (strpos($requestPath, '/stats/users')) {
     import('lib.pkp.api.v1.stats.users.PKPStatsUserHandler');
     return new PKPStatsUserHandler();
+} elseif (strpos($requestPath, '/stats/issues')) {
+    import('api.v1.stats.issues.StatsIssueHandler');
+    return new StatsIssueHandler();
+} elseif (strpos($requestPath, '/stats/contexts')) {
+    import('lib.pkp.api.v1.stats.contexts.PKPStatsContextHandler');
+    return new PKPStatsContextHandler();
+} elseif (strpos($requestPath, '/stats/sushi')) {
+    import('api.v1.stats.sushi.StatsSushiHandler');
+    return new StatsSushiHandler();
 }
```

Comment:
This (and probably other handlers here) needs to handle errors in the default case. Requesting e.g. `http://localhost/git/ojs/index.php/index/api/v1/stats/` causes an error:
```
[20-Jul-2022 01:20:22 UTC] PHP Fatal error:  Uncaught Error: Call to a member function getApp() on int in /home/asmecher/git/ojs-main/lib/pkp/classes/core/APIRouter.inc.php:128
Stack trace:
#0 /home/asmecher/git/ojs-main/lib/pkp/classes/core/Dispatcher.inc.php(163): PKP\core\APIRouter->route()
#1 /home/asmecher/git/ojs-main/lib/pkp/classes/core/PKPApplication.inc.php(379): PKP\core\Dispatcher->dispatch()
#2 /home/asmecher/git/ojs-main/index.php(70): PKP\core\PKPApplication->execute()
#3 {main}
  thrown in /home/asmecher/git/ojs-main/lib/pkp/classes/core/APIRouter.inc.php on line 128
```